qa/rolling-upgrade/src/test/java/org/opensearch/neuralsearch/bwc/AbstractRollingUpgradeTestCase.java

```diff
@@ -130,4 +132,11 @@ protected void createPipelineForSparseEncodingProcessor(String modelId, String p
         );
         createPipelineProcessor(requestBody, pipelineName, modelId);
     }
+
+    protected void createPipelineForTextChunkingProcessor(String pipelineName) throws Exception {
```

Comment:
This seems duplicate code with: https://github.com/opensearch-project/neural-search/pull/661/files#diff-721f0f5596bd046a060235fee514a17ee03d94be58ea10e7d39efc53662647deR106, any chance to make them common?

Reply:
+1, maybe we can have a utility class for common methods like this